### PR TITLE
Update Beatmap Spotlights for Autumn 2020

### PR DESCRIPTION
**Pull request by Icymice**
I've updated the beatmap spotlights wiki page for the Autumn 2020 season as it had not been updated.

### CODE REVIEW

File: wiki/Beatmap_Spotlights/en.md [hunk 1]
```diff
@@ -72,158 +72,84 @@ Each Beatmap Spotlights release is assembled by a dedicated curator team and ind
...
-   - [Hige Driver join. SELEN - Dadadadadadadadadada (qoot8123) \[Oni\]](https://osu.ppy.sh/beatmapsets/237306#taiko/549277)
-   - [Zekk - Falling Down feat. Renko x TRIANGLE (Raiden) \[Hell Oni\]](https://osu.ppy.sh/beatmapsets/1150487#taiko/2401584)
-   - [LeaF - Doppelganger (extended ver.) (Midnaait) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/833092#taiko/1745096)
-- **Playlist D**
-   - [RoughSketch vs. L.E.D.-G - Engoku OVERKILL (Yales) \[Muzukashii\]](https://osu.ppy.sh/beatmapsets/889395#taiko/1860173)
-   - [nmk - sola (TKS) \[Oni\]](https://osu.ppy.sh/beatmapsets/171239#taiko/414274)
-   - [S3RL feat Krystal - R4V3 B0Y (Nishizumi) \[0N1\]](https://osu.ppy.sh/beatmapsets/593010#taiko/1254644)
-   - [Yu_Asahina - Trickstarz (kitafuma) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/1118075#taiko/2335498)
-   - [Nekomata Master+ - encounter (tasuke912) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/474003#taiko/1012645)
+   - [Sakuzyo - Future Coaster (_DUSK_) \[Muzukashii\]](https://osu.ppy.sh/beatmapsets/1056866#taiko/2209098)
```
Comment (zeusminus) [inline wiki/Beatmap_Spotlights/en.md:89]:
```suggestion
   - [Sakuzyo - Future Coaster (\_DUSK\_) \[Muzukashii\]](https://osu.ppy.sh/beatmapsets/1056866#taiko/2209098)
```
escaping

File: wiki/Beatmap_Spotlights/en.md [hunk 2]
```diff
@@ -72,158 +72,84 @@ Each Beatmap Spotlights release is assembled by a dedicated curator team and ind
...
-| Timeshift lobby: Playlist D | 2020-08-28/2020-08-30 |
-| Timeshift lobby: Playlist A and B | 2020-09-04/2020-09-06 |
-| Timeshift lobby: Playlist C and D | 2020-09-11/2020-09-13 |
-| Season conclusion | 2020-09-14/2020-09-18 |
+| Season start | 2020-10-09 |
+| Timeshift lobby: Playlist A | 2020-10-09/2020-10-16 |
+| Timeshift lobby: Playlist B | 2020-10-16/2020-10-23 |
+| Timeshift lobby: Playlist C | 2020-10-23/2020-10-30 |
+| Timeshift lobby: Playlist D | 2020-10-30/2020-11-06 |
+| INTERMISSION | 2020-11-06/2020-11-13 |
```
Comment (zeusminus) [inline wiki/Beatmap_Spotlights/en.md:137]:
```suggestion
| Intermission | 2020-11-06/2020-11-13 |
```
I think sentence case should be used here

File: wiki/Beatmap_Spotlights/en.md [hunk 3]
```diff
@@ -72,158 +72,84 @@ Each Beatmap Spotlights release is assembled by a dedicated curator team and ind
...
-   - [Camellia - POWA OF DA WILDANES (Monheim) \[YATSUKAMIZUOMITSUNO\]](https://osu.ppy.sh/beatmapsets/1098578#mania/2295244)
-   - [Camellia - Beyond the Geostationary Orbit Level (Leniane) \[Equatorial\]](https://osu.ppy.sh/beatmapsets/1011337#mania/2116795)
-- **Playlist D**
-   - [The Flashbulb - Every Two Days (Monheim) \[2\]](https://osu.ppy.sh/beatmapsets/905599#mania/1890121)
-   - [siqlo - One Way Street (Dreamwalker) \[Unilateral\]](https://osu.ppy.sh/beatmapsets/688453#mania/1867902)
-   - [Ta9-5 - Positive (feat. Looci) (Dubstek) \[Loop Loop Loop\]](https://osu.ppy.sh/beatmapsets/1026168#mania/2146057)
-   - [Camellia - PLANET//SHAPER (Evening) \[GEOLOGICAL//IRRESOLUTION\]](https://osu.ppy.sh/beatmapsets/376293#mania/823842)
-   - [Frums - We Want To Run (Blocko) \[Another\]](https://osu.ppy.sh/beatmapsets/1025170#mania/2335802)
+   - [Twenty Knives - Crow's Ghost (Shoegazer) \[Observer\]](https://osu.ppy.sh/beatmapsets/666119#mania/1411678) +HR
+   - [Ekcle - The Impulsive State (Parachor) \[Entropy_\]](https://osu.ppy.sh/beatmapsets/643857#mania/1364765)
```
Comment (zeusminus) [inline wiki/Beatmap_Spotlights/en.md:108]:
```suggestion
   - [Ekcle - The Impulsive State (Parachor) \[Entropy\_\]](https://osu.ppy.sh/beatmapsets/643857#mania/1364765)
```
escaping

**Review by zeusminus — COMMENTED** [review]
might conflict with #4125

**Comment by Icymice** [comment]
> might conflict with #4125

I didn't notice that, hopefully not.

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
did conflict with the other one but this also caught some updates the other one missed. thanks o/

**Pull request merged**